lighthouse-core/scripts/release/prepare-package.sh

```diff
@@ -28,9 +28,6 @@ yarn install
 # Build everything
 yarn build-all
 
-# Package the extension
-node build/build-extension.js package
```

Comment:
I'm confused why we aren't building the extension here anymore? Wouldn't we be building both here now?

Reply:
build-all builds it